MC/config/QC/json/tpc-qc-standard-direct.json

```diff
@@ -3,11 +3,7 @@
     "config": {
       "database": {
         "implementation": "CCDB",
-        "host": "ccdb-test.cern.ch:8080",
-        "username": "not_applicable",
-        "password": "not_applicable",
-        "name": "not_applicable",
-        "maxObjectSize": "20971520"
+        "host": "ccdb-test.cern.ch:8080"
```

Comment:
Hello @lauraser , 
The CCDB should now point to "alice-ccdb.cern.ch", @JianLIUhep .
Can you change it?
Chiara

Reply:
Hi @chiarazampolli @lauraser, we should keep using this host here. See my comment below.

Reply:
We need to change the conditionDB url to `alice-ccdb.cern.ch` at L#26

Reply:
Right, sorry!

Reply:
I updated the code with L#26